docs/tasks.md

```diff
@@ -11,6 +11,7 @@ A list of reusable Cumulus tasks. [Add your own.](adding-a-task.md)
 ## Tasks
 
 ### [@cumulus/discover-granules](https://github.com/nasa/cumulus/tree/master/tasks/discover-granules)
+
```

Comment:
This document is auto-generated by `npm run docs-build` or something. I fixed the script that actually generates it so that the extra lines should be created correctly:

https://github.com/nasa/cumulus/pull/1300/files#diff-56f379556b03830b54f90d82a958d38dR49

Reply:
Heh, got a little too in-a-groove updating lint failures.  Thanks.